generators/fastlane-env/README.md

```diff
@@ -44,3 +44,7 @@ Example:
 
 ### Adding a new device to the iOS provisioning profiles
 The `ios setup --env=<myenv>` task mentionned above has been setup to allow you to regenerate your certificates in case you add a new device. Just run it again to include the new devices.
+
+## Troubleshooting
+
+If 'Cloning GitHub repo' takes too long : the reason might be that you have never initiated connection with GitHub and it is not yet trusted. Enter git clone git@github.com:bamlab/certificates.git and type 'yes' when you are asked if you trust Github.com
```

Comment:
No space before : (`takes too long: the reason [...]`).
Also you can use backtick (`) around git clone git@github.com:bamlab/certificates.git to show that it's code that can be entered :smile: